pkg/storaged/dialog.jsx

```diff
@@ -275,7 +290,71 @@ function is_visible(field, values) {
     return !field.options || field.options.visible == undefined || field.options.visible(values);
 }
 
-const Body = ({ body, fields, values, errors, onChange }) => {
+class Revealer extends React.Component {
+    constructor() {
+        super();
+        this.state = { revealed: false };
+    }
+
+    render() {
+        return (
+            <div>
```

Comment:
Never use an empty <div> (or <span>).

This should either be classed or it should go away, replaced with a `<React.Fragment>` instead.